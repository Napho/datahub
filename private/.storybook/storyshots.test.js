import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  configPath: 'private/.storybook',
  storyKindRegex:/^((?!.*?DontTest).)*$/
});
