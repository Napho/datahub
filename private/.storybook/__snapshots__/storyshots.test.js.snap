// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button With Icon 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  <i
    aria-hidden="true"
    className="right arrow icon"
  />
   
</button>
`;

exports[`Storyshots Button With Icon and Text 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  Continue 
  <i
    aria-hidden="true"
    className="right arrow icon"
  />
   
</button>
`;

exports[`Storyshots Button With Primary Color 1`] = `
<button
  className="ui primary button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  Primary
   
</button>
`;

exports[`Storyshots Button With Secondary Color 1`] = `
<button
  className="ui secondary button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  Secondary
   
</button>
`;

exports[`Storyshots Button With some emoji 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  <span
    aria-label="emojis"
    role="img"
  >
    üòÄ üòé üëç üíØ
  </span>
   
</button>
`;

exports[`Storyshots Button with text 1`] = `
<button
  className="ui button"
  onClick={[Function]}
  tabIndex={undefined}
>
   
  Hello World
   
</button>
`;

exports[`Storyshots Footer Datahub Footer 1`] = `
<div
  className="css-xwxuqh"
>
  <div
    className="css-140jqim"
  >
    <div
      className="css-1tfdju4"
    >
      <div
        className="ui container"
      >
        <div
          className="ui grid"
        >
          <div
            className="row"
          >
            <div
              className="four wide column"
            >
              <h4
                className="css-v73krq"
              >
                First Columns
              </h4>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Data blog 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 2 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 3 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 4 
              </a>
            </div>
            <div
              className="four wide column"
            >
              <h4
                className="css-v73krq"
              >
                Second Columns
              </h4>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Data blog 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 2 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 3 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 4 
              </a>
            </div>
            <div
              className="four wide column"
            >
              <h4
                className="css-v73krq"
              >
                Third Columns
              </h4>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Data blog 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 2 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 3 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 4 
              </a>
            </div>
            <div
              className="four wide column"
            >
              <h4
                className="css-v73krq"
              >
                Fourth Columns
              </h4>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Data blog 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 2 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 3 
              </a>
              <a
                className="css-1j6tgbo"
                href="http://devinit.org/#!/blog"
              >
                 Link 4 
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-8v7vv"
  >
    <div
      className="row"
    >
      <div
        className="ui center aligned container"
      >
        <div
          className="sixteen wide column"
        >
          <a
            className="css-1zicc8"
            href="https://twitter.com/devinitorg"
          >
            <i
              aria-hidden="true"
              className="twitter icon"
            />
          </a>
          <a
            className="css-1zicc8"
            href="https://twitter.com/devinitorg"
          >
            <i
              aria-hidden="true"
              className="facebook f icon"
            />
          </a>
          <a
            className="css-1zicc8"
            href="https://twitter.com/devinitorg"
          >
            <i
              aria-hidden="true"
              className="youtube play icon"
            />
          </a>
          <a
            className="css-1zicc8"
            href="https://twitter.com/devinitorg"
          >
            <i
              aria-hidden="true"
              className="flickr icon"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="lightBlack css-140jqim"
  >
    <div
      className="row"
    >
      <div
        className="ui container"
      >
        <div
          className="css-12p5b9r"
        >
          <p>
            ¬© Development Initiatives 2017. Development Initiatives is the trading name of DI International Ltd. Registered in England and Wales No. 05802543. Development Initiatives Poverty Research is the not-for-profit partner of DI International Ltd. Registered in England and Wales No. 06368740. Read our
            <a
              href="http://devinit.org/wp-content/uploads/2017/01/privacy_and_data_policy.pdf"
            >
              privacy policy
            </a>
            .
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Menu default 1`] = `
<div
  className="ui menu"
/>
`;